bolt-sidecar/src/signer/commit_boost.rs

```diff
@@ -122,9 +123,10 @@ impl CommitBoostSigner {
     /// Sign an object root with the Commit Boost domain.
     pub async fn sign_commit_boost_root(&self, data: [u8; 32]) -> SignerResult<BlsSignature> {
         // convert the pubkey from ethereum_consensus to commit-boost format
-        let pubkey = cb_common::signer::BlsPublicKey::from(
-            alloy::rpc::types::beacon::BlsPublicKey::from_slice(self.pubkey().as_ref()),
-        );
+        // TODO: compat: this parsing is the only way to obtain a Blspubkey. There are no other
+        // possible ways to obtain it without bumping Alloy version
+        let pubkey =
+            cb_common::signer::BlsPublicKey::from_ssz_bytes(self.pubkey().as_ref()).unwrap();
```

Comment:
Extremely annoying

Reply:
As I've written above, can we use `expect` instead of `unwrap`? 

I think writing a quick unit test should clear your doubts, can you do that?